observability/distr_tracing/distr_tracing_tempo/distr-tracing-tempo-installing.adoc

```diff
@@ -6,19 +6,60 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-Installing the {TempoShortName} involves the following steps:
+Installing the {TempoShortName} requires the {TempoOperator} and choosing which type of deployment is best for your use case:
 
-. Setting up supported object storage.
-. Installing the {TempoOperator}.
-. Creating a secret for the object storage credentials.
-. Creating a namespace for a TempoStack instance.
-. Creating a `TempoStack` custom resource to deploy at least one TempoStack instance.
+* For microservices mode, deploy a TempoStack instance in a dedicated OpenShift project.
+* For monolithic mode, deploy a TempoMonolithic instance in a dedicated OpenShift project.
 
-include::modules/distr-tracing-tempo-storage-ref.adoc[leveloffset=+1]
+[IMPORTANT]
+====
+Using object storage requires setting up a supported object store and creating a secret for the object store credentials before deploying a TempoStack or TempoMonolithic instance.
+====
+
+[id="installing-the-tempo-operator"]
+== Installing the {TempoOperator}
+
+You can install the {TempoOperator} by using the web console or the command line.
+
+include::modules/distr-tracing-tempo-install-web-console.adoc[leveloffset=+2]
+
+include::modules/distr-tracing-tempo-install-cli.adoc[leveloffset=+2]
+
+[id="installing-a-tempostack-instance"]
+== Installing a TempoStack instance
+
+You can install a TempoStack instance by using the web console or the command line.
+
+include::modules/distr-tracing-tempo-install-tempostack-web-console.adoc[leveloffset=+2]
+
+include::modules/distr-tracing-tempo-install-tempostack-cli.adoc[leveloffset=+2]
 
-include::modules/distr-tracing-tempo-install-web-console.adoc[leveloffset=+1]
+[id="installing-a-tempomonolithic-instance"]
+== Installing a TempoMonolithic instance
 
-include::modules/distr-tracing-tempo-install-cli.adoc[leveloffset=+1]
+:FeatureName: The TempoMonolithic instance
+include::snippets/technology-preview.adoc[]
+
+You can install a TempoMonolithic instance by using the web console or the command line.
+
+The `TempoMonolithic` custom resource (CR) creates a Tempo deployment in monolithic mode.
+All components of the Tempo deployment, such as the compactor, distributor, ingester, querier, and query frontend, are contained in a single container.
+
+A TempoMonolithic instance supports storing traces in in-memory storage, a Persistent Volume, or object storage.
+
+Tempo deployment in monolithic mode is preferred for a small deployment, demo, testing, and as a migration path of the {JaegerName} all-in-one deployment.
```

Comment:
New to me, but ISG apparently says use "demonstration" 🤷 
```suggestion
Tempo deployment in monolithic mode is preferred for a small deployment, demonstration, testing, and as a migration path of the {JaegerName} all-in-one deployment.
```

Reply:
Thank you. Applied.